pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -88,6 +88,7 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *v1beta1.Knative
 	}
 
 	if manifest == nil {
+		logger.Warnf("No manifest found; no cluster-scoped resources will be finalized")
 		return nil
```

Comment:
Adding at a warn here, otherwise it's very silent op. After previous `"Deleting cluster-scoped resources"` there are no more messages in logs, if `status.manifests` are empty.